core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -99,22 +116,21 @@ private static RocksDB openDB(RocksPageStoreOptions pageStoreOptions, Options ro
    *
    * @param pageStoreOptions options for the rocks page store
    * @param rocksDB RocksDB instance
-   * @param dbOptions the RocksDB options
    */
-  private RocksPageStore(RocksPageStoreOptions pageStoreOptions, RocksDB rocksDB,
-      Options dbOptions) {
-    mPageStoreOptions = pageStoreOptions;
-    mDbOptions = dbOptions;
+  private RocksPageStore(RocksPageStoreOptions pageStoreOptions,
```

Comment:
One great thing about RocksDB JNI is, for everything that extends `RocksObject` you will need to call `close()` on it, for example, double check your `ColumnFamilyHandle`.

Reply:
Thank you for this great tricks!  I just explicitly called close for every RocksObject I can find.